pkg/restapi/did/operation/operations.go

```diff
@@ -145,12 +165,22 @@ func addRequestHeaders(req, newReq *http.Request) {
 	}
 }
 
-func (o *Operation) writeErrorResponse(rw http.ResponseWriter, status int, msg string) {
+func writeOKResponse(rw http.ResponseWriter, status int, contentType string, body []byte) {
```

Comment:
Rename to writeResponse since it takes status code (may not be OK)

Reply:
done